src/components/Tooltip/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+
+const ARROW_OFFSET = '40px';
+const ICON_SIZE = '1rem';
+
+const calcTooltipPosition = (align, width) => {
+  let x = `calc(${width} * -0.5 + 50%);`;
+
+  if (align === 'left') {
+    x = `calc(-1 * ${ARROW_OFFSET})`;
+  }
+  if (align === 'right') {
+    x = `calc(${width} * -1 + ${ARROW_OFFSET})`;
+  }
+
+  return `left: ${x};`;
+};
+
+const calcArrowPosition = (align, width, offset = '0px') => {
+  let x = `calc(50% + ${offset} - ${ICON_SIZE} * 0.5);`;
+
+  if (align === 'left') {
+    x = `calc(${ARROW_OFFSET} + ${offset});`;
+  }
+  if (align === 'right') {
+    x = `calc(${width} - ${ARROW_OFFSET} + ${offset});`;
+  }
+
+  return `left: ${x};`;
+};
+
+const Container = styled.div`
+  position: relative;
+
+  &:hover .tooltip {
+    visibility: visible;
+    opacity: 1;
+  }
+`;
+
+const Icon = styled(FontAwesomeIcon)`
+  color: ${props => props.theme.brandBlue};
+  width: 1rem;
+`;
+
+const TooltipContainer = styled.div`
+  position: absolute;
+  padding: ${props => props.theme.mSpacing};
+  border-radius: 6px;
+
+  visibility: hidden;
+  text-align: center;
+
+  width: ${props => props.width};
+
+  background-color: #d1e3fe;
```

Comment:
Let's use the same `background-color` value that we are using for secondary buttons on hover, just to maintain consistency. Apologies if the mockups didn't match this.

Reply:
👍